sms_log_handler/providers/africastalking/base.py

```diff
@@ -0,0 +1,21 @@
+from sms_log_handler.providers.base import SMSProviderBase
+from sms_log_handler.providers.africastalking.gateway import AfricasTalkingGateway  # noqa
+
+
+class AfricasTalkingProvider(SMSProviderBase):
+    """
+    AfricasTalking provider
+    """
+
+    def send(self, phone_numbers, message):
+        """
+        Handles sending of the error message to the provider phone numbers
+
+        params:
+            phone_numbers: An array of phonenumbers to sms
+            message: This are the contents of the message that is being sent.
+
+        """
+        phone_numbers_csv = ','.join(phone_numbers)
+        gateway = AfricasTalkingGateway(self.key, self.secret)
+        gateway.send_message(phone_numbers_csv, message)
```

Comment:
There is no exception that will be raised if for example the phone numbers are wrong format?